Bus Scheduler/src/shared/components/molecules/inputWithIcon/index.tsx

```diff
@@ -0,0 +1,25 @@
+import '@/shared/components/atoms/input/style.scss';
+import '@/shared/components/atoms/input';
+import Input, { InputType } from '@/shared/components/atoms/input';
```

Comment:
will it be good to use some other names than molecules/organisms ?

Reply:
Keeping as it is for now, as we discussed